events/app.py

```diff
@@ -69,7 +68,6 @@ def build_event_info(info, time):
 @app.route('/v1/edit/<event_id>', methods=['PUT'])
 def edit_event(event_id):
     """Edit the event with the given id."""
-    pass
```

Comment:
This is a handy hint that the code isn't implemented yet; I would leave it.

Reply:
Added back in ea7f1b5